client-py/SessionExample.py

```diff
@@ -280,6 +280,51 @@
     while session_data_set.has_next():
         print(session_data_set.next())
 
+# insert string records of one device
+time_list = [1, 2, 3]
+measurements_list = [
+    ["s_01", "s_02", "s_03"],
+    ["s_01", "s_02", "s_03"],
+    ["s_01", "s_02", "s_03"],
+]
+values_list = [["False", "22", "33"], ["True", "1", "23"], ["False", "15", "26"]]
+
+session.insert_string_records_of_one_device(
+    "root.sg_test_01.d_03",
+    time_list,
+    measurements_list,
+    values_list,
+)
+
+# insert aligned string record into the database.
+time_list = [1, 2, 3]
+measurements_list = [
+    ["s_01", "s_02", "s_03"],
+    ["s_01", "s_02", "s_03"],
+    ["s_01", "s_02", "s_03"],
+]
+values_list = [["False", "22", "33"], ["True", "1", "23"], ["False", "15", "26"]]
+session.insert_aligned_string_records_of_one_device(
+    "root.sg_test_01.d_04",
+    time_list,
+    measurements_list,
+    values_list,
+)
```

Comment:
Move aligned example to SessionAlignedTimeseriesExample.py?

Reply:
Ok, it has been modified as your suggestion